test/e2e/gardener/common.go

```diff
@@ -15,7 +15,6 @@ import (
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
 	"github.com/gardener/gardener/pkg/utils/timewindow"
-	"github.com/gardener/gardener/test/framework"
```

Comment:
Can you add an `.import-restrictions` file to `test/e2e` that prevents importing `test/{framework,testmachinery}` again?